bot/cogs/grades_data/grades_cog.py

```diff
@@ -277,5 +338,37 @@ async def grades(self, ctx, course, year=MIN_YEAR):
 
         await ctx.send(embed=embed)
 
+    @commands.command()
+    async def prof(self, ctx, firstName, lastName, detailed = 'yes'):
```

Comment:
we should make it so it only needs a last name and require the user to specify if it finds duplicate of that name

Reply:
detailed should be a boolean converter with a default value, why would we want it to not be detailed anyway?

Reply:
Your first comment sounds like a issue that should be fixed by someone else. I'm not 100% sure why because if they're using this bot, they likely know the professor's name.

Your second comment: Some people would just want to see pass (A-C) vs fail (<D) rate. Why not have the option?

Reply:
fair enough detailed, as for first name, i dont know the first name of any of professors tbh lol. its not a well known thing

Reply:
Ideally this is a tool you'd use when you know what professor you want to take, not before.

You can get professor a few names via the grades cog anyway